### ISSUE TITLE
Keystatic showing blank screen when trying to load admin ui locally

### ISSUE THREAD

**Issue by Tdnshah** [Thinkmill/keystatic#782]
I was trying to all keystatic cms to my existing Astro based website, I followed the documentation as available here https://keystatic.com/docs/installation-astro

so after completing all the step in the documentation, when I visit https://127.0.0.1:4321/keystatic I just get a blank page, so would like to know if this is a bug or am I missing any additional step to be followed for adding keystatic in exisiting site.

Astro version : 3.0.6

**Comment by stefanprobst** [comment]
are you starting astro with `astro dev --host 127.0.0.1`? (see https://keystatic.com/docs/installation-astro#running-keystatic)

**Comment by Tdnshah** [comment]
Yeah I do, I have followed all the steps mentioned in this docs https://keystatic.com/docs/installation-astro#running-keystatic

**Comment by Tdnshah** [comment]
I could resolve this myself, some wrong json array structure was added me causing an error, now able to load config and even able to see the keystatic admin UI thanks. :)
